python/felis/tap.py

```diff
@@ -375,20 +471,45 @@ def visit_constraint(self, constraint_obj: Constraint) -> tuple:
         return key, key_columns
 
     def visit_index(self, index_obj: Index, table_obj: Table) -> None:
+        """Visit an index object and update the TAP_SCHEMA representation.
+
+        Parameters
+        ----------
+        index_obj
+            The index object to visit.
+        table_obj
+            The table object which the index belongs to.
+        """
         columns = [self.graph_index[col_id] for col_id in getattr(index_obj, "columns", [])]
         # if just one column and it's indexed, update the object
         if len(columns) == 1:
             columns[0].indexed = 1
         return None
 
-    def _schema_name(self, schema_name: str | None = None) -> str | None:
+    def _schema_name(
+        self, schema_name: str | None = None
+    ) -> str | None:  # DM-44870: Usage of this method needs to be better understood and possibly removed
+        """Return the schema name.
+
+        Parameters
+        ----------
+        schema_name
+            Name of the schema.
+        """
```

Comment:
missing Returns? Or is it that this is a private method?

Reply:
Added.